ui/src/sandbox.rs

```diff
@@ -542,6 +567,7 @@ mod test {
     fn basic_functionality() {
         let req = ExecuteRequest {
             channel: Channel::Stable,
+            crate_type: CrateType::Binary,
```

Comment:
Oops, I guess I don't run these tests in CI at all! I should fix that...

Reply:
On my local test sandbox::test::memory_usage_is_limited is failing with:

```rust
---- sandbox::test::memory_usage_is_limited stdout ----
thread 'sandbox::test::memory_usage_is_limited' panicked at 'assertion failed: resp.stderr.contains("Killed")', sandbox.rs:848:8
```
Not sure if that's a real issue, or just differences between the containers on my machine and production.